index.html

```diff
@@ -293,6 +293,141 @@ <h1>Paweł Nowicki</h1>
+                <div class="col-md-6 col-lg-5 text-center text-md-left mb-4 mb-md-0">
+                    <h6 class="mb-0">Dołącz do nas w portalach społecznościowych!</h6>
+                </div>
+                <!-- Grid column -->
+
+                <!-- Grid column -->
+                <div class="col-md-6 col-lg-7 text-center text-md-right">
+
+                    <!-- Facebook -->
+                    <a class="fb-ic">
+                        <i class="footer-facebook"> </i>
+                        <a href="https://www.facebook.com/infosharepl/?ref=br_rs" target="_blank"><img src="img/facebook23.png" alt=""></a>
+                    </a>
+                    <!-- Twitter -->
+                    <a class="tw-ic">
+                        <i class="footer-facebook"> </i>
+                        <a href="" target="_blank"><img src="img/twitter30.png" alt=""></a>
+                    </a>
+                    <!--Instagram-->
+                    <a class="ins-ic">
+                        <i class="footer-instagram"> </i>
+                        <a href="https://www.instagram.com/?hl=pl" target="_blank"><img src="img/instagram30.png" alt=""></a>
+                    </a>
+
+                </div>
+                <!-- Grid column -->
+
+            </div>
+            <!-- Grid row-->
+
+        </div>
+    </div>
+
+    <!-- Footer Links -->
+    <div class="container text-center text-md-left mt-5">
+
+        <!-- Grid row -->
+        <div class="row mt-3">
+
+            <!-- Grid column -->
+            <div class="col-md-3 col-lg-4 col-xl-3 mx-auto mb-4">
+
+                <!-- Content -->
+                <h6 class="footer-content">C-BULA</h6>
+                <hr class="deep-purple accent-2 mb-4 mt-0 d-inline-block mx-auto" style="width: 60px;">
+                <p> Już dziś zrób zakupy w C-buli,<br> a zaoszczędzisz czas i pieniądze.</p>
+
+            </div>
+            <!-- Grid column -->
+
+            <!-- Grid column -->
+            <div class="col-md-2 col-lg-2 col-xl-2 mx-auto mb-4">
+
+                <!-- Links -->
+                <h6 class="footer-shops">Sklepy</h6>
+                <hr class="deep-purple accent-2 mb-4 mt-0 d-inline-block mx-auto" style="width: 60px;">
+                <p>
+                    <a href="#!">Spożywczy</a>
+                </p>
+                <p>
+                    <a href="#!">Pralnia</a>
+                </p>
+                <p>
+                    <a href="#!">Szewc</a>
+                </p>
+                <p>
+                    <a href="#!">Pasmanteria</a>
+                </p>
+
+            </div>
+            <!-- Grid column -->
+
+            <!-- Grid column -->
+            <div class="col-md-3 col-lg-2 col-xl-2 mx-auto mb-4">
+
+                <!-- Links -->
+                <h6 class="footer-useful-links">Przydatne</h6>
+                <hr class="deep-purple accent-2 mb-4 mt-0 d-inline-block mx-auto" style="width: 60px;">
```

Comment:
a te stylowanie nie powinno być w css?

Reply:
Jeśli dobrze zrozumiałam Kamila to mozna zostawić to stylkowanie w Html.